src/scripts/OSFramework/OSUI/Pattern/AnimatedLabel/scss/_animated-label.scss

```diff
@@ -177,3 +175,28 @@
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.animated-label-text {
+		font-weight: var(--font-semi-bold);
+		padding: 0 10px;
```

Comment:
Why do we need to change things other than colors and/or opacity? 

Also, if we really need to, OSUI Design Systems uses a scale of 8, so 10 should never be a value, it should be 8 or 12 and using the existing --space variables.